decidim-core/lib/decidim/gamification/badge.rb

```diff
@@ -27,7 +27,6 @@ class Badge
       attribute :reset, Proc
 
       validates :name, :levels, presence: true
-      validates :levels, empty: false
```

Comment:
Just leaving a note to myself: this check is already part of the `presence` validator, so the empty validation is redundant.